cmd/dao_tests/pubkey_test.go

```diff
@@ -63,7 +75,126 @@ func TestSimple(t *testing.T) {
 	assert.Nil(t, nil)
 }
 
+func TestCreatePubkey(t *testing.T) {
+	CleanUpDatabase(t)
+	pkDao, ctx, err := Setup(t, "Create pubkey")
+	if err != nil {
+		t.Errorf("Database setup had failed: %v", err)
+		return
+	}
+	err = pkDao.Create(ctx, createPk())
+	if err != nil {
+		t.Errorf("Create pubkey test had failed: %v", err)
+		return
+	}
+
+	pubkeys, err := pkDao.List(ctx, 10, 0)
+	if err != nil {
+		t.Errorf("Create pubkey test had failed: %v", err)
+		return
+	}
+
+	assert.Equal(t, 1, len(pubkeys), "Create pubkey test had failed.")
+}
+
+func TestListPubkey(t *testing.T) {
+	CleanUpDatabase(t)
+	pkDao, ctx, err := Setup(t, "List pubkey")
+	if err != nil {
+		t.Errorf("Database setup had failed: %v", err)
+		return
+	}
+	err = pkDao.Create(ctx, createPk())
+	pubkeys, err := pkDao.List(ctx, 100, 0)
+	if err != nil {
+		t.Errorf("List pubkey test had failed: %v", err)
+		return
+	}
+	assert.Equal(t, 1, len(pubkeys), "List Pubkey error.")
+}
+
+func TestUpdatePubkey(t *testing.T) {
+	CleanUpDatabase(t)
+	updatePk := &models.Pubkey{
+		ID:        1,
+		AccountID: 1,
+		Name:      "avitova-ed25519-2021",
+		Body:      "ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIEhnn80ZywmjeBFFOGm+cm+5HUwm62qTVnjKlOdYFLHN avitova",
+	}
+	pkDao, ctx, err := Setup(t, "Update pubkey")
+	if err != nil {
+		t.Errorf("Database setup had failed. %s", err)
+		return
+	}
+	err = pkDao.Create(ctx, createPk())
+	err = pkDao.Update(ctx, updatePk)
+	if err != nil {
+		t.Errorf("Update pubkey test had failed. %s", err)
+		return
+	}
+
+	pubkeys, err := pkDao.List(ctx, 10, 0)
+	if err != nil {
+		t.Errorf("Update pubkey test had failed. %s", err)
+		return
+	}
+	assert.Equal(t, updatePk.Name, pubkeys[0].Name, "Update pubkey test had failed.")
+}
+
+func TestGetPubkeyById(t *testing.T) {
+	CleanUpDatabase(t)
+	pkDao, ctx, err := Setup(t, "Get pubkey")
+	if err != nil {
+		t.Errorf("Database setup had failed. %s", err)
+		return
+	}
+	err = pkDao.Create(ctx, createPk())
+	if err != nil {
+		t.Errorf("Delete pubkey test had failed. %s", err)
+		return
+	}
+	pubkey, err := pkDao.GetById(ctx, 1)
+	if err != nil {
+		t.Errorf("Get pubkey test had failed.")
+		return
+	}
+	assert.Equal(t, "lzap-ed25519-2021", pubkey.Name, "Get Pubkey error: pubkey name does not match.")
+	assert.Equal(t, "ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIEhnn80ZywmjeBFFOGm+cm+5HUwm62qTVnjKlOdYFLHN lzap", pubkey.Body, "Get Pubkey error: pubkey body does not match.")
+
+}
+
+func TestDeletePubkeyById(t *testing.T) {
+	CleanUpDatabase(t)
+	pkDao, ctx, err := Setup(t, "Delete pubkey")
+	if err != nil {
+		t.Errorf("Database setup had failed")
+		return
```

Comment:
I'm not sure, but shouldn't `t.Errorf` stop the test execution? Do we need to do it manually by `return`?